pkg/operator/configobservation/apiserver/observer_apiserver.go

```diff
@@ -0,0 +1,262 @@
+package apiserver
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
+)
+
+func ObserveUserClientCABundle(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	listers := genericListers.(configobservation.Listers)
+	var errs []error
+
+	// will always return an empty map
+	emptyConfig := map[string]interface{}{}
+
+	destinationLocation := resourcesynccontroller.ResourceLocation{
+		Name:      "user-client-ca",
+		Namespace: operatorclient.OperatorNamespace,
+	}
+
+	// zero-valued source location will cause destination to be deleted
+	sourceLocation := resourcesynccontroller.ResourceLocation{}
```

Comment:
if you made this a pointer, you just just check nil-ness instead of a bool, right?